test/Dapr.AspNetCore.IntegrationTest.App/DaprController.cs

```diff
@@ -25,22 +25,22 @@ public ActionResult<UserInfo> RegisterUser(UserInfo user)
             return user; // echo back the user for testing
         }
 
-        [HttpPost("/controllerwithoutstateentry/{widget}")]
-        public async Task AddOneWithoutStateEntry([FromServices]StateClient state, [FromState] Widget widget)
+        [HttpPost("/controllerwithoutstateentry/{testStore}/{widget}")]
```

Comment:
You don't need to add a parameter to the route, it's not actually doing anything because you're not parsing the state store name from the URL. You should be able to revert just this part of the test.